packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -25,17 +28,20 @@ class FramesBarChart extends CoreElement {
     add(frameUIgraph);
 
     // Make sure DIV exist.
-    // TODO(terry): Use requestAnimationFrame instead?
-    Timer.run(() {
-      if (!_createdPlot)
-        _createdPlot = frameUIgraph.createPlot(timelineController);
+    frameBarChartStateMixin.setState(() {
+      if (!_createdPlot) {
+        frameUIgraph.createPlot(timelineController);
+        _createdPlot = true;
+      }
     });
 
     timelineController.onFrameAdded.listen((TimelineFrame frame) {
       frameUIgraph.process(timelineController, frame);
     });
   }
 
+  SetStateMixin frameBarChartStateMixin = SetStateMixin();
```

Comment:
have this class mixin the minxin instead of having a separate mixin object.

Reply:
Okay, I used this because that's how our code uses SetStateMixin.